recipes/open62541/all/conanfile.py

```diff
@@ -0,0 +1,300 @@
+        self._cmake.definitions["UA_ENABLE_EXPERIMENTAL_HISTORIZING"] = self.options.enable_experimental_historizing
+        self._cmake.definitions["UA_ENABLE_PUBSUB"] = self.options.enable_pubsub
+        self._cmake.definitions["UA_ENABLE_PUBSUB_ETH_UADP"] = self.options.enable_pubsub_eth_uadp
+        self._cmake.definitions["UA_ENABLE_PUBSUB_DELTAFRAMES"] = self.options.enable_pubsub_deltaframes
+        self._cmake.definitions["UA_ENABLE_PUBSUB_INFORMATIONMODEL"] = self.options.enable_pubsub_informationmodel
+        self._cmake.definitions["UA_ENABLE_PUBSUB_INFORMATIONMODEL_METHODS"] = self.options.enable_pubsub_informationmodel_methods
+        self._cmake.definitions["UA_ENABLE_PUBSUB_CUSTOM_PUBLISH_HANDLING"] = self.options.enable_pubsub_custom_publish_handling
+        self._cmake.definitions["UA_ENABLE_JSON_ENCODING"] = self.options.enable_json_encoding
+        self._cmake.definitions["UA_ENABLE_STATUSCODE_DESCRIPTIONS"] = self.options.enable_statuscode_descriptions
+        self._cmake.definitions["UA_ENABLE_NODESET_COMPILER_DESCRIPTIONS"] = self.options.enable_nodeset_compiler_descriptions
+        self._cmake.definitions["UA_ENABLE_MALLOC_SINGLETON"] = self.options.enable_malloc_singleton
+        self._cmake.definitions["UA_MSVC_FORCE_STATIC_CRT"] = self.options.msvc_force_static_crt
+        self._cmake.definitions["UA_ENABLE_DISCOVERY_SEMAPHORE"] = self.options.enable_discovery_semaphore
+        self._cmake.definitions["UA_DEBUG_DUMP_PKGS"] = self.options.debug_dump_pkgs
+        self._cmake.definitions["UA_ENABLE_HARDENING"] = self.options.enable_hardening
+        self._cmake.definitions["OPEN62541_VERSION"] = self.version
+        self._cmake.definitions["UA_DEBUG" ] = self.settings.build_type == "Debug"
+        self._cmake.definitions["UA_ENABLE_WEBSOCKET_SERVER"] = self.options.with_libwebsockets
+        self._cmake.definitions["UA_ENABLE_SUBSCRIPTIONS_ALARMS_CONDITIONS"] = self.options.enable_subscriptions_alarms_conditions
+        self._cmake.definitions["UA_ENABLE_PARSING"] = self.options.enable_parsing
+        self._cmake.definitions["UA_ENABLE_DA"] = self.options.enable_da
+        self._cmake.definitions["UA_ENABLE_PUBSUB_ETH_UADP_XDP"] = self.options.enable_pubsub_eth_uadp_xdp
+        self._cmake.definitions["UA_ENABLE_PUBSUB_ETH_UADP_ETF"] = self.options.enable_pubsub_eth_uadp_etf
+        self._cmake.definitions["UA_ENABLE_PUBSUB_MQTT"] = self.options.enable_pubsub_mqtt
+        self._cmake.definitions["UA_FORCE_32BIT"] = not(self.settings.arch == "armv8" or self.settings.arch == "x86_64")
+        self._cmake.definitions["UA_FORCE_CPP"] = self.options.force_cpp
+        self._cmake.definitions["UA_ENABLE_TYPEDESCRIPTION"] = self.options.enable_typedescription
+        self._cmake.definitions["UA_NAMESPACE_ZERO"] = self.options.namespace_zero
+
+        self._cmake.definitions["UA_BUILD_EXAMPLES"] = False
+        self._cmake.definitions["UA_BUILD_UNIT_TESTS"] = False
```

Comment:
this option also defaults to Off in [CmakeLists.txt](https://github.com/open62541/open62541/blob/master/CMakeLists.txt#L405), thus again, no need to set it